### PR TITLE
Position of MSH 15-5-02, MSH 15-52, HESS J1514-591

### PR DESCRIPTION
**Pull request by cdeil**
I wonder if we have the correct position for the source that goes by the names of MSH 15-5-02, MSH 15-52, HESS J1514-591 in gamma-cat.

It's this one:
* http://gamma-sky.net/#/cat/tev/79
* http://gamma-sky.net/#/map?target=320.278,-1.073&fov=4.286&marker=snrcat
* https://github.com/gammapy/gamma-cat/blob/31d9a8f039d2ff52f913b85ef6cc16b3e45a36b0/output/gammacat.yaml#L6637

The current position we have is this:
```
  - ra: 228.32080078125
  - dec: -59.08169937133789
  - pos_ra: 228.52915954589844
  - pos_dec: -59.157501220703125
```
The `ra/dec` comes from the SIMBAD position for MSH 15-5-02 from [input/sources/tev-000079.yaml](https://github.com/gammapy/gamma-cat/blob/31d9a8f039d2ff52f913b85ef6cc16b3e45a36b0/input/sources/tev-000079.yaml#L23):
```
pos:
  simbad_id: MSH 15-5-02
  ra: 228.3208
  dec: -59.0817
```
The `pos_ra/pos_dec` comes from the  HESS J1514-591 measurement in [gamma-cat/input/data/2005/2005A%26A...435L..17A/tev-000079.yaml](https://github.com/gammapy/gamma-cat/blob/b651de8d1d793e924764ffb13c8ec189bce9ea7d/input/data/2005/2005A%2526A...435L..17A/tev-000079.yaml#L9)
```
pos:
  ra: {val: 15h14m7s, err: 0h0m21s, err_sys: 20s}
  dec: {val: -59d9m27s, err: 11s, err_sys: 20s}
```
I double-checked the values in the paper. That is the correct base-fit position from the elongated Gaussian fit in http://adsabs.harvard.edu/abs/2005A%26A...435L..17A .

I'm aware the situation is complex and that there is https://arxiv.org/abs/1709.01422, but I still wonder if the positions we have now in `gamma-cat`, but also if the ones given in SIMBAD make sense or not.

---

Here are all relevant positions:
```
>>> from astropy.coordinates import SkyCoord
>>> msh1 = SkyCoord.from_name('MSH 15-5-02')
>>> msh1
<SkyCoord (ICRS): (ra, dec) in deg
    (228.320833, -59.0816667)>
>>> msh2 = SkyCoord.from_name('MSH 15-52')
>>> msh2
<SkyCoord (ICRS): (ra, dec) in deg
    (228.529167, -59.1575)>
>>> hess_paper = SkyCoord('15h14m7s -59d9m27s')
>>> hess_paper
<SkyCoord (ICRS): (ra, dec) in deg
    (228.52916667, -59.1575)>
>>> hess_simbad = SkyCoord.from_name('HESS J1514-591')
>>> hess_simbad
<SkyCoord (ICRS): (ra, dec) in deg
    (228.320833, -59.0816667)>
>>> psr = SkyCoord.from_name('PSR B1509-58')
>>> psr
<SkyCoord (ICRS): (ra, dec) in deg
    (228.4775, -59.1375)>
>>> snr = SkyCoord.from_name('SNR G320.3-01.1')
>>> snr
<SkyCoord (ICRS): (ra, dec) in deg
    (228.320833, -59.0816667)>
```

The position of `HESS J1514-591` in the paper and in SIMBAD doesn't agree:
```
>>> hess_paper.separation(hess_simbad)
<Angle 0.1310878 deg>
```

The `hess_simbad` position matches `msh1` and `snr`:
```
>>> hess_simbad.separation(msh1)
<Angle 0. deg>
>>> hess_simbad.separation(snr)
<Angle 0. deg>
```

The `hess_paper` position matches `msh2` almost perfectly:
```
>>> hess_paper.separation(msh2)
<Angle 1.70893319e-07 deg>
```

The separation between those two positions is 0.13 deg:
```
>>> msh1.separation(msh2)
<Angle 0.13108794 deg>
```
As expected, neither coincides with the pulsar:
```
>>> msh1.separation(psr)
<Angle 0.09791187 deg>
>>> msh2.separation(psr)
<Angle 0.03319725 deg>
```

I'm not able to find the entry for `msh2` in SIMBAD directly, this resolves to `msh1`!?
http://simbad.u-strasbg.fr/simbad/sim-id?Ident=HESS+J1514-591

---

Bottom line: I think we should consider changing common_name from `MSH 15-5-02` to `MSH 15-52` in gamma-cat, or at least change the position in `input/sources/tev-000079.yaml` from `msh1` to `msh2`.

Thoughts?

### CODE REVIEW

**Comment by cdeil** [comment]
I've made a commit with changes here: acf4947
I'm reasonably happy with this, so merging now.
Discussion via email with Michelle, Yves and Konrad is still ongoing, if it concludes in a different way or someone else speaks up here we can always make another commit.

I did change the `common_name` to "MSH 15-52", putting the previous "MSH 15-5-02" under `other_names`. It seems that "MSH 15-52" is the "modern name" used in Fermi-LAT and HESS papers. From the original MSH catalog paper it isn't clear what the correct name should be, they do use "15-52" with an italic "2" sometimes, sometimes there is a space between the 5 and 2.

For the position, I did put the best-fit position from  http://adsabs.harvard.edu/abs/2005A%26A...435L..17A  , avoiding the issue concerning complex morphology / counterpart catalog positions for this source. I also checked the upcoming HGPS paper that contains a 3-Gauss morphology model. The centroid is almost identical with the one from  2005A%26A...435L..17A, i.e. what we have now in gamma-cat.

What we can't change is the CTA first data challenge, which used gamma-cat. It will forever have MSH 15-52 at ~0.1 offset from the true TeV source position. Oh well. :-)
See [here](https://github.com/gammapy/gammapy-extra/blob/b54a6f715e3bc7e61ab67476890822a928ebf202/notebooks/cta_1dc_analysis_msh15-52.ipynb)
